include/particle.h

```diff
@@ -179,9 +198,24 @@ class Particle : public ParticleBase<Tdim> {
     this->stress_.col(phase) = stress;
   }
 
+  //! Initial pore pressure
+  //! \param[in] pore_fluid Index corresponding to fluid phase
+  //! \param[in] pore pressure Initial pore pressure
+  void initial_pore_pressure(const unsigned pore_fluid,
```

Comment:
No need to `const` variables that are size `double` or below because it makes it less efficient. Pass those by value.